libbeat/docs/step-configure-output.asciidoc

```diff
@@ -32,12 +32,12 @@ output.elasticsearch:
 ----------------------------------------------------------------------
 ifndef::has_module_steps[]
 +
-ifndef::only-elasticsearch[]
+ifndef::no_logstash_output[]
 To send output to {ls}, 
 <<logstash-output,Configure the {ls} output>> instead. For all other
 outputs, see <<configuring-output>>.
-endif::only-elasticsearch[]
+endif::[]
 ifdef::only-elasticsearch[]
```

Comment:
Did you mean to leave in this `ifdef::only-elasticsearch[]`?

Reply:
Nope...not sure why I missed that in my search. I'll fix.

Reply:
Looking more closely, I realize I left this intentionally for the situation where we might have a Beat that only supports ES output. However, I think I can make this more generic and avoid keeping this attribute, so I'll do that.